LibraryManagementSystem/Forms/ViewTab.cs

```diff
@@ -124,35 +124,35 @@ private void txtSearch_KeyDown(object sender, KeyEventArgs e)
         // Sort feature incomplete, currently in testing
         private void cmbSort_SelectedIndexChanged(object sender, EventArgs e)
         {
-            con = new SqlConnection("Data Source=" + Program.globalServer + "\\SQLEXPRESS;Initial Catalog=libraryData;Integrated Security=True");
+            con = new SqlConnection("Data Source=" + Program.globalServer + "\\SQLEXPRESS;Initial Catalog=LibDat;Integrated Security=True");
             con.Open();
             string cmdText;
 
             switch (cmbSort.SelectedIndex)
             {
                 // Sort Title in alphabetical order
                 case 0:
-                    cmdText = "SELECT * FROM booksData ORDER BY Title";
+                    cmdText = "SELECT * FROM book ORDER BY title";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort Author in alphabetical order
                 case 1:
-                    cmdText = "SELECT * FROM booksData ORDER BY Author";
+                    cmdText = "SELECT * FROM book ORDER BY author";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort BookID in ascending order
                 case 2:
-                    cmdText = "SELECT * FROM booksData ORDER BY BookID";
+                    cmdText = "SELECT * FROM book ORDER BY book_id";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort books by the date they were borrowed
                 case 3:
-                    cmdText = "SELECT * FROM booksData ORDER BY DateBorrowed";
+                    cmdText = "SELECT * FROM borrow_data ORDER BY borrow_date";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort books by the date they should be returned
                 case 4:
-                    cmdText = "SELECT * FROM booksData ORDER BY DateToReturn";
+                    cmdText = "SELECT * FROM booksData ORDER BY return_date";
```

Comment:
Same case here as previous comment, but adding <kbd>ORDER BY return_date</kbd> instead. But tbh I think if we fix this I think it would still have the same results as sorting by borrow_date since all return_date values are just borrow_date + 7 days (1 week)